### ISSUE TITLE
Delete this repo

### ISSUE THREAD

**Issue by Devamitta** [Devamitta/online-tool-for-db#5]
@falko-strenzke I think I am going to delete this repo, since we already have a [tool for maintaining DPD](https://github.com/digitalpalidictionary/dpd-db). What do you think?

**Comment by falko-strenzke** [comment]
> @falko-strenzke I think I am going to delete this repo, since we already have a [tool for maintaining DPD](https://github.com/digitalpalidictionary/dpd-db). What do you think?

Yes, it's fine for me if you delete the repo. Out of curiosity: how does your collaboration work with https://github.com/digitalpalidictionary/dpd-db ? Are you somehow using GitHub as the basis of working together on the dictionary data?

**Comment by Devamitta** [comment]
Mainly, we use separate databases and work independently. Once a month, Bhante Bodhirasa has a tool that applies my corrections and additions. Afterward, I update the ID for those additions in my backup and rebuild the database from the combined backup TSV. And we again synced. And up to next mouth we again work independently.

**Comment by falko-strenzke** [comment]
Good to hear that you have found a suitable way to cooperate on the dictionary. I saved the latest version of my document in case we should need anything from there later. So, bhante, please feel free to go ahead and delete this repo.
